fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/filter/SelectorFilter.java

```diff
@@ -20,7 +21,19 @@
      * @param value  Value of the filter
      */
     public SelectorFilter(Dimension dimension, String value) {
-        super(dimension, DefaultFilterType.SELECTOR);
+        this(dimension, value, null);
```

Comment:
how about calling super that takes 2 args?

Reply:
right